### PR TITLE
Update multinode-related content in the architecture section

### PR DESCRIPTION
**Pull request by erimatnor**
This change updates multinode-related content of the architecture
section with new terminology (focusing on distribute hypertables) and
small tweaks.

### CODE REVIEW

File: introduction/architecture.md [hunk 1]
```diff
@@ -110,53 +96,52 @@ while maintaining support for multiple indexes.
...
+consists of multiple databases stored across one or more TimescaleDB
+instances. A database that is part of a distributed database can
+assume one of two *node* roles (but not both):
 
-The two roles that nodes can take on are:
 1) Access node
 2) Data node
 
-A access node accepts all requests from the client, distributes the requests and queries
+A client connects to an access node database. It then distributes the requests and queries
```
Comment (mkindahl) [inline introduction/architecture.md]: The "It" here can refer to either the subject or the object. It looks like it is referring to the client, not the access node.

File: introduction/architecture.md [hunk 2]
```diff
@@ -110,53 +96,52 @@ while maintaining support for multiple indexes.
...
 hypertables, as well as regular PostgreSQL tables.
 
-Data nodes store data associated with distributed hypertables. Data nodes do not store cluster-wide
-information, and otherwise look just as if they were single TimescaleDB instances. Avoid
-directly accessing local hypertables or chunks of a distributed hypertable on a data node
-as the action can fail or make the distributed hypertable inconsistent. There are
-very few reasons to ever access a data node directly, and all accesses should be done at
-the access node level.
+Data nodes store data associated with a distribute database (i.e., the
+distributed hypertables within it). Data nodes do not store
```
Comment (mkindahl) [inline introduction/architecture.md]: Suggestion: "Data nodes store the data of the distributed database." You have already mentioned that a distributed database is a collection of nodes above, so I think this is redundant.

File: introduction/architecture.md [hunk 3]
```diff
@@ -110,53 +96,52 @@ while maintaining support for multiple indexes.
...
+cluster-wide information, and otherwise look just as if they were
+stand-alone TimescaleDB instances. Avoid directly accessing local
+hypertables or chunks of a distributed hypertable on a data node as
+the action can fail or make the distributed hypertable
+inconsistent. There are very few reasons to ever access a data node
```
Comment (mkindahl) [inline introduction/architecture.md]: Suggestion: "You should not directly access hypertables or chunks on data nodes. Doing so might lead to inconsistent distributed hypertables."

File: introduction/architecture.md [hunk 4]
```diff
@@ -110,53 +96,52 @@ while maintaining support for multiple indexes.
...
-### Distributed Hypertables
+purposes, look just like a single instance of TimescaleDB from an operational perspective.
 
-As discussed above, distributed databases contain distributed hypertables. Distributed hypertables are essentially
-the same as regular hypertables, except that they have an additional layer of abstraction where
-access nodes must be able to logically manage distributed hypertables. Distributed hypertables
-are also composed of chunks.
+### Configuring Distributed Hypertables
 
 With distributed hypertables, we suggest partitioning the hypertables both by space and by time.
```
Comment (mkindahl) [inline introduction/architecture.md]: Given the description below, I do not think this is a suggestion. It is how you need to partition to actually utilize the distributed nature of the distributed hypertable, so something like "to utilize the full capacity of distributed hypertable, you should partition the hypertable in both the space and time dimension."

File: introduction/architecture.md [hunk 5]
```diff
@@ -110,62 +96,62 @@ while maintaining support for multiple indexes.
...
 
-### Node Types
+A distributed hypertable exist in a *distributed database* that
+consists of multiple databases stored across one or more TimescaleDB
+instances. A database that is part of a distributed database can
+assume one of two *node* roles (but not both):
 
-The two roles that nodes can take on are:
 1) Access node
 2) Data node
```
Comment (mkindahl) [inline introduction/architecture.md]: Nit: Lists are usually hard to read unless it consists of three items or more, so suggesting "...can be either an *access node* or a *data node*."

File: introduction/architecture.md [hunk 6]
```diff
@@ -110,62 +96,62 @@ while maintaining support for multiple indexes.
...
+are also some features of regular hypertables that distributed
+hypertables do not support (see section on current limitations).
 
-TimescaleDB supports multi-node clustering by leveraging the hypertable and chunk primitives described above.
-In the multi-node topology, all nodes are TimescaleDB instances, but may play different roles
-depending on their responsibilities within a distributed database.
+### Distributed Databases and Nodes
 
-### Node Types
+A distributed hypertable exist in a *distributed database* that
```
Comment (solugebefola) [inline introduction/architecture.md]:
```suggestion
A distributed hypertable exists in a *distributed database* that
```

File: introduction/architecture.md [hunk 7]
```diff
@@ -110,62 +96,62 @@ while maintaining support for multiple indexes.
...
+(i.e., a cluster) by leveraging the same hypertable and chunk
+primitives as described above. This allows TimescaleDB to scale
+inserts and queries beyond the capabilities of a single TimescaleDB
+instance.
 
->:WARNING: Running TimescaleDB in a multi-node setup is currently in PRIVATE BETA.
-This method of deployment is not meant for production use. For more information,
-please
-[contact us][contact].
+Distributed hypertables look very similar to regular hypertables, with
```
Comment (solugebefola) [inline introduction/architecture.md]:
```suggestion
Distributed hypertables and regular hypertables look very similar, with
```

File: introduction/architecture.md [hunk 8]
```diff
@@ -110,62 +96,62 @@ while maintaining support for multiple indexes.
...
+the main difference being that chunks are not stored locally. There
```
Comment (solugebefola) [inline introduction/architecture.md]:
```suggestion
the main difference being that distributed chunks are not stored locally. There
```

File: introduction/architecture.md [hunk 9]
```diff
@@ -110,62 +96,62 @@ while maintaining support for multiple indexes.
...
-
-With distributed hypertables, we suggest partitioning the hypertables both by space and by time.
-If you only partition data by time, that chunk will have to fill up before the access node chooses
-another data node to store the next chunk, and thus during that chunk's time interval, all writes
-to the latest interval will be handled by a single data node, rather than load balanced across all
-available data nodes. On the other hand, if you specify a space partition, the access node
-will distribute chunks across multiple data nodes based on the space partition so that multiple
-chunks are created for a given chunk time interval, and both reads and writes to that recent time
-interval will be load balanced across the cluster.
+purposes, look just like a single instance of TimescaleDB from an operational perspective.
```
Comment (solugebefola) [inline introduction/architecture.md]: not sure if this suggestion is technically accurate, but:
```suggestion
purposes, act just like a single instance of TimescaleDB from an operational perspective.
```

File: introduction/architecture.md [hunk 10]
```diff
@@ -110,62 +96,62 @@ while maintaining support for multiple indexes.
...
+
+### Configuring Distributed Hypertables
+
+To ensure best performance, you should partition a distributed
+hypertable by both time and a space. If you only partition data by
```
Comment (solugebefola) [inline introduction/architecture.md]:
```suggestion
hypertable by both time and space. If you only partition data by
```

File: introduction/architecture.md [hunk 11]
```diff
@@ -110,62 +96,62 @@ while maintaining support for multiple indexes.
...
+time, that chunk will have to fill up before the access node chooses
+another data node to store the next chunk, and thus during that
```
Comment (solugebefola) [inline introduction/architecture.md]:
```suggestion
another data node to store the next chunk, so during that
```

File: introduction/architecture.md (hunk 5 above)
Comment (erimatnor) [reply introduction/architecture.md]: I'll keep for now since there before. Think docs probably needs a larger overhaul at some point.

**Pull request merged**